services/crates/crates-dependents.service.js

```diff
@@ -0,0 +1,49 @@
+import Joi from 'joi'
+import { pathParams } from '../index.js'
+import { metric } from '../text-formatters.js'
+import { nonNegativeInteger } from '../validators.js'
+import { BaseCratesService, description } from './crates-base.js'
+
+const dependentsResponseSchema = Joi.object({
```

Comment:
Dependents of a crate is served under `/api/v1/crates/:crate/reverse_dependencies`. The schema is specific to this service, that's why I thought it'd be better if it stayed in this module rather than `crates-base.js`.

Reply:
agreed :+1: